lib/sinon/util/core/called-in-order.js

```diff
@@ -1,15 +1,34 @@
 "use strict";
 
+var every = Array.prototype.every;
+
 module.exports = function calledInOrder(spies) {
+    var callMap = {};
+
+    function hasCallsLeft(spy) {
+        if (callMap[spy.id] === undefined) {
+            callMap[spy.id] = 0;
+        }
+
+        return callMap[spy.id] < spy.callCount;
+    }
+
     if (arguments.length > 1) {
-        spies = arguments;
+        spies = Array.prototype.slice.call(arguments);
```

Comment:
MINOR: The only reason to use `slice` here would be if you plan to used `spies.every(...)` later. However you are using `every.call(...)` so you can revert this line.